src/components/PaddedContainer/index.tsx

```diff
@@ -40,6 +41,6 @@ const StyledContainer = styled('div')<PaddedContainerProps>(
   `,
 );
 
-export const PaddedContainer: React.FC<PaddedContainerProps> = props => {
+export const PaddedContainer: ReactFC<PaddedContainerProps> = props => {
```

Comment:
This looks too similar to the React.FC that it may cause confusion. Why not just extend PaddedContainerProps with the new props?

Reply:
Changed it.